tests/test_xmad_calendar.py

```diff
@@ -38,10 +38,11 @@ def test_holidays_fall_on_weekend(self):
             # open on both the prior Friday and the following Monday.
             pd.Timestamp('2010-04-30', tz='UTC'),
             pd.Timestamp('2010-05-03', tz='UTC'),
+
```

Comment:
minor thing, but could you modify this commit's wording to be `through 2010` rather than `until 2010 inclusive`? the commit's wording makes it seem like Christmas Eve and New Year's Eve are not observed in 2010

Reply:
Done